saffron/src/lib.rs

```diff
@@ -1103,92 +1130,108 @@ impl Cron {
     /// assert_eq!(cron.next_after(date), date.with_minute(10));
     /// ```
     #[inline]
-    pub fn next_after(&self, date: DateTime<Utc>) -> Option<DateTime<Utc>> {
-        let date = date.trunc_subsecs(0).with_second(0).unwrap();
+    pub fn next_after(&self, dt: DateTime<Utc>) -> Option<DateTime<Utc>> {
+        let dt = next_minute(minute_floor(dt))?;
         if self.any() {
-            self.find_next(date)
-        } else {
-            None
-        }
-    }
-
-    const MUST_CONTAIN_MINUTE: &'static str = "Expression must contain at least one minute";
-
-    fn next_minute(time: NaiveTime) -> Option<NaiveTime> {
-        if time.minute() < 59 {
-            Some(NaiveTime::from_hms(time.hour(), time.minute() + 1, 0))
-        } else if time.hour() < 23 {
-            Some(NaiveTime::from_hms(time.hour() + 1, 0, 0))
+            self.find_next(dt, chrono::MAX_DATETIME)
         } else {
             None
         }
     }
 
-    fn find_next(&self, dt: DateTime<Utc>) -> Option<DateTime<Utc>> {
+    /// Finds the next (current inclusive) matching date time in the future within the specified
+    /// date time bound, or none if the search exceeds the bound.
+    fn find_next(&self, dt: DateTime<Utc>, bound: DateTime<Utc>) -> Option<DateTime<Utc>> {
```

Comment:
`dt` isn't a very useful name, because all the parameters are datetimes. Maybe `start` would be a better name, that way readers know how it's being used differently to `bound`? I think this comment applies to a lot of the uses of `dt: DateTime<_>` as a parameter in this codebase.